get_pdfs.go

```diff
@@ -25,34 +27,47 @@ func initConfig() {
 	if err != nil {
 		log.Fatal()
 	}
+
+	clientCSV = client.Client{
+		APIKey:     CONFIG.PDFTABLES_API_KEY,
+		HTTPClient: http.DefaultClient,
+	}
+}
+
+func folderCheck() {
```

Comment:
make this take the directory so that it can be called with any folder we want to check